metadata/InvoicesAustralia__2__metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-  "changeLog": "v20.8 release",
+  "changeLog": "Release v2020.8",
   "cpu": 0,
   "description": "Machine Learning model for extracting commonly occurring data points from Australian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
-  "displayName": "AustralianInvoices",
+  "displayName": "InvoicesAustralia",
```

Comment:
"name" and "displayName" should match no? "name" also needs to change to "InvoicesAustralia"